gulpfile.js

```diff
@@ -38,6 +38,30 @@ function parseCommandLine() {
 }
 
 const karmaOptions = parseCommandLine();
+const { run } = require('./scripts/gulp/run');
+
+/**
+ * Task to output a draft changelog to be appended to CHANGELOG.md at the
+ * top of the file. Remove git logs that are not relevant for an consumer
```

Comment:
I suggest to remove the part after the first sentence and just refer the reader to `docs/releases.md`. This avoids the comment getting out of sync with the releases.md docs.

Reply:
I agree